README.md

```diff
@@ -1,48 +1,120 @@
-# README
+* `# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Fortytude is an object oriented based website for the Temple University Libraries.
 
-Things you may want to cover:
+## System Requirements
 
-* Ruby version
+- Ruby 2.5.1
+- Google OmniAuth2
```

Comment:
Is this a System Requirement? There is nothing special thay need to install, correct?

Reply:
When you do this on the Mac under RBEnv, it's hard to tell. I'll try provisioning it for a VM know for sure.

Reply:
I just mean Google Omniauth. Not ruby.

Reply:
✔️